.travis.yml

```diff
@@ -0,0 +1,35 @@
+# testing file for Travis
+# https://github.com/neuropoly/ivado-medical-imaging
+language: python
+python:
+  - "3.6"
+  - "3.7"
+matrix:
+  include:
+    - os: linux  # list of Linux env: https://docs.travis-ci.com/user/reference/overview/
+      name: "Ubuntu 18.04 (Bionic Beaver)"
+      dist: bionic
+    - os: linux
+      name: "Ubuntu 16.04 (Xenial)"
+      dist: xenial
+    - os: linux
+      name: "Ubuntu 14.04 (Trusty)"
+      dist: trusty
+    #- os: osx
```

Comment:
any reason to remove osx?

Reply:
Seems that most **OSX** distributions on Travis don't have python 3.X by default (waiting for Catalina...). For some unknown reasons, _xcode9.4_ works with python3 but tests return a timeout error while **Linux** tests pass.

Trying to fix this. Any help is welcomed

Reply:
> tests return a timeout error
i would start looking at where is the lag (which test?) and then try to reduce it. Travis has a maximum running time for processes.